analyzers/.sonarlint/SonarAnalyzer.slconfig

```diff
@@ -8,8 +8,8 @@
   "ProjectName": "SonarAnalyzer .Net",
   "Profiles": {
     "CSharp": {
-      "ProfileKey": "AWT63_RpV-oLu4FdMlyI",
-      "ProfileTimestamp": "2022-07-22T07:07:51Z"
+      "ProfileKey": "AYS9H5b7szY8h8f_RKYR",
```

Comment:
This change is expected, as I had to rearrange profiles a little.

At the same time, what value this update brings? Is it required for the connected mode to work? Or is there any other problem on the dev machine?

Reply:
> Or is there any other problem on the dev machine?

It gives you a small warning in VS that the SonarLint profile needs to be updated and once you do it you get these changes. It might bother others while working on a PR.